site/reference/fn/eval/README.md

```diff
@@ -101,42 +101,42 @@ fn-args:
 ```shell
 # execute container my-fn on the resources in DIR directory and
 # write output back to DIR
-$ kpt fn --image gcr.io/example.com/my-fn eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn  
 ```
 
 ```shell
 # execute container my-fn on the resources in DIR directory with
 # `functionConfig` my-fn-config
-$ kpt fn --image gcr.io/example.com/my-fn --fn-config my-fn-config eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --fn-config my-fn-config  
 ```
 
 ```shell
 # execute container my-fn with an input ConfigMap containing `data: {foo: bar}`
-$ kpt fn --image gcr.io/example.com/my-fn:v1.0.0 eval DIR -- foo=bar
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn:v1.0.0   -- foo=bar
 ```
 
 ```shell
 # execute executable my-fn on the resources in DIR directory and
 # write output back to DIR
-$ kpt fn --exec-path ./my-fn eval DIR
+$ kpt fn eval DIR --exec-path ./my-fn  
 ```
 
 ```shell
 # execute container my-fn on the resources in DIR directory,
 # save structured results in /tmp/my-results dir and write output back to DIR
-$ kpt fn --image gcr.io/example.com/my-fn --results-dir /tmp/my-results-dir eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --results-dir /tmp/my-results-dir  
 ```
 
 ```shell
 # execute container my-fn on the resources in DIR directory with network access enabled,
 # and write output back to DIR
-$ kpt fn --image gcr.io/example.com/my-fn --network eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --network 
 ```
 
 ```shell
 # execute container my-fn on the resource in DIR and export KUBECONFIG
 # and foo environment variable
-$ kpt fn --image gcr.io/example.com/my-fn --env KUBECONFIG -e foo=bar eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --env KUBECONFIG -e foo=bar  
```

Comment:
Can you pl. run `make generate` to re-generate the Go files that `kpt fn eval --help()` uses. That will  keep `kpt fn eval --help()` in sync with these changes.

Reply:
The following commit made these changes
* Removing of the trailing spaces
* Updating the go files by `make generate`